.github/workflows/post-release-qa.yml

```diff
@@ -0,0 +1,45 @@
+name: Post Release SDK Integration Tests
+
+on: [workflow_dispatch,push]
+
+jobs:
+  verify-cocoapods-iOS:
+    runs-on: macos-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Install pod, build project and run tests
+        run: |
+            cd  SDKIntegrationTestApps/PostRelease-iOSReleaseTest-Cocoapods/
+            pod install
+            xcodebuild test -scheme iOSReleaseTest -workspace iOSReleaseTest.xcworkspace -destination 'platform=iOS Simulator,name=iPhone 13,OS=latest' | xcpretty && exit ${PIPESTATUS[0]}
```

Comment:
Minor note. At some point the sim model we picked here will be no longer included by default with Xcode. We can either install it, or maybe make a script to list out available and pick the newest.